src/v2/Apps/Partner/partnerRoutes.tsx

```diff
@@ -180,9 +180,38 @@ export const partnerRoutes: AppRouteConfig[] = [
           ) {
             partner(id: $partnerId) @principalField {
               ...Works_partner @arguments(input: $input)
+              displayWorksSection
+              counts {
+                eligibleArtworks
+              }
             }
           }
         `,
+        render: ({ Component, props, match }) => {
+          if (!(Component && props)) {
+            return
+          }
+
+          const { partner } = props as any
```

Comment:
Is it possible to get rid of this `any`?

Looks like you could also shorten this to `if (!props.partner) ...` which might avoid this as well.

Reply:
![Screen Shot 2021-05-24 at 4 33 44 PM](https://user-images.githubusercontent.com/79980131/119355333-cf681100-bcad-11eb-99df-de1f878f02c2.png)

This way wouldn't avoid the problem. The render props can't find somehow the partner field, that's why we decided avoiding the issue with `as any`

Reply:
@damassi do you know if there's a way to provide types for the `render` function?